src/main/java/core/basesyntax/serviceimpl/FileWriterImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+
+    @Override
+    public void writeToFile(String fileName, String data) {
+        try {
+            Files.write(Paths.get(fileName), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file by path - " + fileName, e);

```

Comment:
Excellent job on rethrowing the exception as a RuntimeException with an informative message. This follows the checklist guidance on exception handling.